File: ServerJava/src/main/java/com/company/UserDetails.java
package com.company;

public class UserDetails {
    private int Id;
    private String Fname;
    private String Lname;
    private String UserName;
    private String Mail;
    private String Password;
    private int Mode;
    private String Picture;

    public UserDetails( int Id,String Fname, String Lname, String UserName,String Mail, String Password,int Mode,String Picture)
    {
        this.setId(Id);
        this.setFname(Fname);
        this.setLname(Lname);
        this.setUserName(UserName);
        this.setMail(Mail);
        this.setPassword(Password);
        this.setMode(Mode);
        this.setPicture(Picture);
    }

    // Getter Methods

    public int getId() {
        return Id;
    }

    public String getFname() {
        return Fname;
    }

    public String getLname() {
        return Lname;
    }

    public String getUserName() {
        return UserName;
    }

    public String getMail() {
        return Mail;
    }

    public String getPassword() {
        return Password;
    }

    public int getMode() {
        return Mode;
    }

    // Setter Methods

    public void setId(int Id) {
        this.Id = Id;
    }

    public void setFname(String Fname) {
        this.Fname = Fname;
    }

    public void setLname(String Lname) {
        this.Lname = Lname;
    }

    public void setUserName(String UserName) {
        this.UserName = UserName;
    }

    public void setMail(String Mail) {
        this.Mail = Mail;
    }

    public void setPassword(String Password) {
        this.Password = Password;
    }

    public void setMode(int Mode) {
        this.Mode = Mode;
    }

    public String getPicture() {
        return Picture;
    }

    public void setPicture(String picture) {
        Picture = picture;
    }
}

File: ClientCsharp/ClientCsharp/App_Code/UserDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientCsharp.App_Code
{
    public class UserDetails
    {
        private int id;
        private string fname;
        private string lname;
        private string userName;
        private string mail;
        private string password;
        private int mode;
        private string picture;

        public UserDetails()
        {
        }
        public UserDetails(int id, string fname, string lname, string userName, string mail, string password, int mode,string picture)
        {
            this.id = id;
            this.fname = fname;
            this.lname = lname;
            this.userName = userName;
            this.mail = mail;
            this.password = password;
            this.mode = mode;
            this.picture = picture;
        }

        public int Id
        {
            get { return this.id; }
            set { this.id = value; }
        }
        public string Fname
        {
            get { return this.fname; }
            set { this.fname = value; }
        }
        public string Lname
        {
            get { return this.lname; }
            set { this.lname = value; }
        }
        public string UserName { get => userName; set => userName = value; }
        public string Mail { get => mail; set => mail = value; }
        public string Password { get => password; set => password = value; }
        public int Mode { get => mode; set => mode = value; }
        public string Picture { get => picture; set => picture = value; }
    }
}
